mongodbatlas/x509_authentication_database_users.go

```diff
@@ -34,7 +34,7 @@ type UserCertificate struct {
 	MonthsUntilExpiration int    `json:"monthsUntilExpiration,omitempty"` // A number of months that the created certificate is valid for before expiry, up to 24 months.default 3.
 	Certificate           string `json:"certificate,omitempty"`
 
-	ID        string `json:"_id,omitempty"`       // Serial number of this certificate.
+	ID        int    `json:"_id,omitempty"`       // Serial number of this certificate.
```

Comment:
Could you please change it to *int64  to avoid integer overflow errors

Reply:
okay thanks 👍